### ISSUE TITLE
[Autofix error] RUF100 noqa with additional comment

### ISSUE THREAD

**Issue by araffin** [charliermarsh/ruff#3623]
Command: `ruff --select RUF test.py --fix`

Minimal code:
```python
from torch import nn as nn  # noqa: F401 pylint: disable=unused-import
```

Ruff version: 0.0.257

Side question: what is the recommended way to deal with multiple "ignore"? (for instance `noqa` for ruff and `type: ignore` when using `mypy`)

**Comment by JonathanPlasse** [comment]
You should separate the multiple "ignore" by `#` (c.f. https://til.simonwillison.net/python/ignore-both-flake8-and-mypy)
```python
from torch import nn as nn  # noqa: F401 # pylint: disable=unused-import
```
A lot of tools need the `#` for it to work.
```python
# Ruff does not detect the noqa on the following line
id: int = ""  # pyright: ignore noqa: A001
# Pyright does not detect the ignore on the following line
id: int = ""  # noqa: A001 pyright: ignore
# These work as expected
id: int = ""  # pyright: ignore # noqa: A001
id: int = ""  # noqa: A001 # pyright: ignore
```

**Comment by charliermarsh** [comment]
This is fixed on `main`, I think (#3589).

**Comment by charliermarsh** [comment]
(But yeah, using the extra hash will also fix it.)
